package.json

```diff
@@ -5,26 +5,26 @@
     "node": ">=10"
   },
   "devDependencies": {
-    "@babel/core": "7.7.4",
-    "@babel/plugin-transform-runtime": "7.7.4",
-    "@babel/preset-env": "7.7.4",
-    "@babel/runtime": "7.7.4",
+    "@babel/core": "7.7.7",
+    "@babel/plugin-transform-runtime": "7.7.6",
```

Comment:
Why update this one version behind?

Reply:
No idea, it's the latest [published version](https://www.npmjs.com/package/@babel/plugin-transform-runtime).